public/css/main.css

```diff
@@ -0,0 +1,934 @@
+    -webkit-transform: rotate(75deg);
+       -moz-transform: rotate(75deg);
+        -ms-transform: rotate(75deg);
+         -o-transform: rotate(75deg);
+            transform: rotate(75deg);
+    -webkit-transform-origin: 100% 100%;
+       -moz-transform-origin: 100% 100%;
+        -ms-transform-origin: 100% 100%;
+         -o-transform-origin: 100% 100%;
+            transform-origin: 100% 100%;
+}
+.white-notch:before, .white-notch:after {
+  background-color: #fff;
+  z-index: 1;
+}
+.black-notch:before, .black-notch:after {
+  background-color: #2a2a2e;
+}
+@media screen and (min-width: 50em) {
+  .orange-notch:before,
+  .orange-notch:after, .white-notch:before,
+  .white-notch:after, .black-notch:before,
+  .black-notch:after {
+    bottom: -38px;
+  }
+}
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    Top navigation bar
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+.top-nav {
+  padding: auto 0;
+}
+.top-nav li {
+  text-align: center;
+  border: 1px solid rgba(0,0,0,0.2);
+  padding: .5em;
+  font-weight: 300;
+}
+/* flex nav wrapper - adapted from flexboxin5.com */
+.flex-navbar {
+  display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+  display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+  display: -ms-flexbox;      /* TWEENER - IE 10 */
+  display: -webkit-flex;     /* NEW - Chrome */
+  display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
+  -webkit-flex-direction: row;
+  -webkit-flex-wrap: wrap;
+  -webkit-justify-content: center;
+  -webkit-align-content: flex-start;
+  -webkit-align-items: flex-start;
+}
+/* flex nav items - adapted from flexboxin5.com */
+.flex-nav-1, .flex-nav-2, .flex-nav-3, .flex-nav-4, .flex-nav-5, .flex-nav-6 {
+  -webkit-flex-grow: 1;
+  -webkit-flex-shrink: 0;
+  -webkit-flex-basis: 8em;
+  flex-grow: 1;
+  flex-shrink: 0;
+  flex-basis: 8em;
+}
+/* Nav desktop layout */
+@media screen and (min-width: 50em) {
+  .top-nav {
+    padding-left: 10em;
+    padding-right: 10em;
+    border-bottom: 1px solid rgba(124,131,135,0.5);
+  }
+  .top-nav li {
+    border: none;
+  }
+  /* flex nav items - adapted from flexboxin5.com */
+  .flex-nav-1, .flex-nav-2, .flex-nav-3, .flex-nav-4, .flex-nav-5, .flex-nav-6 {
+    -webkit-flex-basis: 4em;
+    flex-basis: 4em;
+  }
+}
+/*&&&&&&&&&&&&&&&&
+
+    Header
+
+&&&&&&&&&&&&&&&&&&*/
+header {
+  clear: left;
+  position: relative;
+  background-image: url("../images/top-bkgrnd.jpg");
+  /* TA Dale said a relative path like this, inside our css file, was fine */
+  background-position: top;
+  background-repeat: no-repeat;
+  height: 15em;
+  text-align: center;
+  padding-bottom: 1em;
+  -webkit-align-items: center;
+          align-items: center;
+  -webkit-justify-content: center;
+          justify-content: center;
+}
+.icon-droplet {
+  margin-bottom: .3em;
+  display: inline-block;
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 50%;
+  background-color: #fd6632;
+  font-size: 3.157em;
+  color: #fff;
+  line-height: 1.5;
```

Comment:
You could also achieve the same visual effect with padding, instead of width/height/line-height:

```
.icon-droplet {
    margin-bottom: .3em;
    display: inline-block;
    /* width: 1.5em; */
    /* height: 1.5em; */
    border-radius: 50%;
    background-color: #fd6632;
    font-size: 3.157em;
    color: #fff;
    /* line-height: 1.5; */
    padding: 0.25em; /* For example */
}
```